external-crates/move/crates/move-analyzer/trace-debug/package.json

```diff
@@ -1,156 +1,179 @@
 {
-  "name": "move-trace-debug",
```

Comment:
Most changes in this file are due to me finally auto-formatting it. Important ones are marked explicitly